gp-templates/translation-row-editor-meta.php

```diff
@@ -53,113 +53,118 @@
-		<dl>
-			<dt><?php _e( 'Date added (GMT):', 'glotpress' ); ?></dt>
-			<dd id="gmt-date-added-<?php echo esc_attr( $translation->row_id ); ?>"><?php echo esc_html( $translation->translation_added ); ?></dd>
-		</dl>
-		<dl>
-			<dt><?php _e( 'Date added (local):', 'glotpress' ); ?></dt>
-			<dd id="local-date-added-<?php echo esc_attr( $translation->row_id ); ?>"><?php _e( 'Calculating...', 'glotpress' ); ?></dd>
-		</dl>
-	<?php endif; ?>
-	<?php if ( $translation->user ) : ?>
-		<dl>
-			<dt><?php _e( 'Translated by:', 'glotpress' ); ?></dt>
-			<dd><?php gp_link_user( $translation->user ); ?></dd>
-		</dl>
-	<?php endif; ?>
-	<?php if ( $translation->user_last_modified && ( ! $translation->user || $translation->user->ID !== $translation->user_last_modified->ID ) ) : ?>
-		<dl>
-			<dt>
-			<?php
+	$meta_sidebar .= apply_filters( 'gp_original_extracted_comments', $translation->extracted_comments, $translation );
+	$meta_sidebar .= '			</dd>';
+	$meta_sidebar .= '		</dl>';
+}
+if ( $translation->translation_added && '0000-00-00 00:00:00' !== $translation->translation_added ) {
+	$meta_sidebar .= '		<dl>';
+	$meta_sidebar .= '			<dt>' . __( 'Date added (GMT):', 'glotpress' ) . '</dt>';
+	$meta_sidebar .= '			<dd id="gmt-date-added-' . esc_attr( $translation->row_id ) . '">' . esc_html( $translation->translation_added ) . '</dd>';
+	$meta_sidebar .= '		</dl>';
+	$meta_sidebar .= '		<dl>';
+	$meta_sidebar .= '			<dt>' . __( 'Date added (local):', 'glotpress' ) . '</dt>';
+	$meta_sidebar .= '			<dd id="local-date-added-' . esc_attr( $translation->row_id ) . '">' . __( 'Calculating...', 'glotpress' ) . '</dd>';
+	$meta_sidebar .= '		</dl>';
+}
+if ( $translation->user ) {
+	$meta_sidebar .= '		<dl>';
+	$meta_sidebar .= '			<dt>' . __( 'Translated by:', 'glotpress' ) . '</dt>';
+	$meta_sidebar .= '			<dd>' . gp_link_user_get( $translation->user ) . '</dd>';
+	$meta_sidebar .= '		</dl>';
+}
+
+if ( $translation->user_last_modified && ( ! $translation->user || $translation->user->ID !== $translation->user_last_modified->ID ) ) {
+	$meta_sidebar .= '		<dl>';
+	$meta_sidebar .= '			<dt>';
 			if ( 'current' === $translation->translation_status ) {
-				_e( 'Approved by:', 'glotpress' );
+				$meta_sidebar .= __( 'Approved by:', 'glotpress' );
 			} elseif ( 'rejected' === $translation->translation_status ) {
-				_e( 'Rejected by:', 'glotpress' );
+				$meta_sidebar .= __( 'Rejected by:', 'glotpress' );
 			} else {
-				_e( 'Last updated by:', 'glotpress' );
+				$meta_sidebar .= __( 'Last updated by:', 'glotpress' );
 			}
-			?>
-			</dt>
-			<dd><?php gp_link_user( $translation->user_last_modified ); ?></dd>
-		</dl>
-	<?php endif; ?>
-	<?php references( $project, $translation ); ?>
+	$meta_sidebar .= '			</dt>';
+	$meta_sidebar .= '			<dd>' . gp_link_user_get( $translation->user_last_modified ) . '</dd>';
+	$meta_sidebar .= '		</dl>';
+}
+ob_start();
+references( $project, $translation );
+$meta_sidebar .= ob_get_clean();
```

Comment:
Can this be escaped with `wp_kses()`?

Reply:
Done [here](https://github.com/GlotPress/GlotPress/pull/1531/commits/f4b49283fb5e75029bdcd74778a5b65ed2e918d1).